src/blockchain.rs

```diff
@@ -46,10 +46,3 @@ pub fn is_valid_chain(chain: & Blockchain) -> bool{
 	}
 	return true
 }
-
-pub fn resolve<'a>(curr_chain: &'a Blockchain, new_chain: &'a Blockchain) -> &'a Blockchain {
```

Comment:
```
   Compiling blockchain v0.1.0 (file:///home/localsys/projects/test/blockchain)
error[E0495]: cannot infer an appropriate lifetime for lifetime parameter 'a in function call due to conflicting requirements
  --> src/server.rs:41:26
   |
41 |         let new_blocks = resolve(self.blockchain, &blocks);
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
note: first, the lifetime cannot outlive the anonymous lifetime #1 defined on the body at 36:43...
  --> src/server.rs:36:44
   |
36 | 	fn blocks(&self, data: MsgData) -> String {
   | 	                                          ^
note: ...so that reference does not outlive borrowed content
  --> src/server.rs:41:34
   |
41 |         let new_blocks = resolve(self.blockchain, &blocks);
   |                                  ^^^^^^^^^^^^^^^
note: but, the lifetime must be valid for the lifetime 'a as defined on the body at 36:43...
  --> src/server.rs:36:44
   |
36 | 	fn blocks(&self, data: MsgData) -> String {
   | 	                                          ^
note: ...so that reference does not outlive borrowed content
  --> src/server.rs:42:27
   |
42 |         self.blockchain = &mut *new_blocks;
   |                           ^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `blockchain`.

To learn more, run the command again with --verbose.
```